src/areas/mod.rs

```diff
@@ -31,14 +31,14 @@ impl Area {
 #[derive(Clone)]
 pub struct Areas {
     areas: Vec<Area>,
-    touch_width: u32,
-    touch_height: u32,
+    touch_width: i32,
+    touch_height: i32,
```

Comment:
what was the rationale for switching from unsigned -> signed ints?

Reply:
Just to make it more consistent. Since all (or at least most?) other integers seem to be `i32`. This results in fewer casts.